lua/csharp/features/user-secrets/init.lua

```diff
@@ -0,0 +1,95 @@
+local M = {}
+local logger = require("csharp.log")
+local utils = require("csharp.utils")
+local dotnet_cli = require("csharp.modules.dotnet-cli")
+
+-- Creates the user secret file if the same
+-- doesn't exists
+local function _ensure_secret_exists(user_secret_file_path)
+  local file, _ = io.open(user_secret_file_path, "r+")
+
+  if not file then
+    os.execute("mkdir -p " .. user_secret_file_path)
```

Comment:
This line will create a directory matching the file path (e.g., ~/.microsoft/user-secrets/{some guid}/secrets.json/) even if the string ends with file extensions (e.g., .json).

Reply:
Hey @iabdelkareem did some code changes, could you check again? Thanks for the reply about the directory, I missed it. Now I am creating the directory if it doesn't exists and then creating the file with "{ }" as you suggested as well!